Mathlib/Dynamics/Flow.lean

```diff
@@ -84,6 +84,8 @@ structure Flow (τ : Type*) [TopologicalSpace τ] [AddMonoid τ] [ContinuousAdd
   map_add' : ∀ t₁ t₂ x, toFun (t₁ + t₂) x = toFun t₁ (toFun t₂ x)
   map_zero' : ∀ x, toFun 0 x = x
 
+attribute [fun_prop] Flow.cont'
```

Comment:
This ta is necessary, so `continuous` below can be proven by fun_prop.

Reply:
I realize that, but the implicit question was not "is it necessary for something else in this PR?" but rather "is it ever going to be useful beyond that?". If the answer to the latter question was "no" (e.g., because it's always used in curried form with the result `Flow.continuous` below), then I would argue we shouldn't tag `Flow.cont'` with `fun_prop`.